member-service/src/main/java/com/hedvig/personservice/persons/web/PersonController.kt

```diff
@@ -79,4 +80,10 @@ class PersonController @Autowired constructor(
         val personFlags = personService.getAllPersonFlags(person)
         return ResponseEntity.ok(personFlags)
     }
+
+    @PostMapping("/has/signed")
```

Comment:
I think this endpoint is more descriptive
```suggestion
    @PostMapping("/hasPersonSigned")
```

Reply:
or 
```suggestion
    @PostMapping("/hasPersonBeenSigned")
```

Reply:
Since it is "/person" on the top I felt like this was pretty neat,  maybe "hasSigned"?

Reply:
Totaly missed that sorry about that